packages/esm-outpatient-app/src/overlay.scss

```diff
@@ -0,0 +1,71 @@
+@import './root.scss';
+
+.desktopOverlay {
+  position: sticky;
+  top: $spacing-09;
+  height: calc(100vh - 3rem);
+  min-width: 40rem;
+  border-left: 1px solid $text-03;
+  overflow: hidden;
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+}
+
+.tabletOverlay {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 9999;
+  background-color: $ui-01;
+  overflow: hidden;
+  padding-top: $spacing-09;
+  display: grid;
+  grid-template-rows: 1fr auto;
+}
```

Comment:
@denniskigen The amount of CSS/SCSS in our project makes me uneasy. Is this doing anything special? Is there a way we can standardize some of this using esm-styleguide? Ideally this PR should involve almost no custom styling.

(I'm not going to block this PR on us getting our styling situation figured out, of course, but I want to start the conversation).